swri_transform_util/include/swri_transform_util/earth_constants.h

```diff
@@ -37,6 +37,9 @@ namespace swri_transform_util
    */
   static const double _earth_equator_radius = 6378137.0;
 
+  /**
+   * Earth mean radius
```

Comment:
A comment that just echoes the name of the variable is not very useful.  Maybe also say what units it's in?